cleanlab/object_detection/rank.py

```diff
@@ -410,16 +435,16 @@ def _get_min_possible_similarity(
 
 
 def _get_valid_inputs_for_compute_scores_per_image(
-    *,
     alpha: float,
+    *,
```

Comment:
why was this shifted below alpha?

Reply:
alpha wasn't optional in the code

Reply:
SGTM. Note we also don't really need to worry about the usage of * and positional vs non-positional arguments for internal methods FYI. That distinction is just important for user-facing methods.

Reply:
aah thanks, I was about to ask that question